drools-wb-services/drools-wb-verifier/drools-wb-verifier-backend/src/test/java/org/drools/workbench/services/verifier/webworker/client/testutil/TestUtil.java

```diff
@@ -19,136 +19,106 @@
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.io.Reader;
+import java.util.HashSet;
+import java.util.List;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 import org.drools.workbench.services.verifier.api.client.reporting.ExplanationProvider;
 import org.drools.workbench.services.verifier.api.client.reporting.Issue;
 import org.drools.workbench.services.verifier.webworker.client.DecisionTableAnalyzerFromFileTest;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.fail;
+
+import static org.assertj.core.api.Assertions.assertThat;
 
 public class TestUtil {
 
-    public static String loadResource( final String name ) throws
-                                                           Exception {
-        final InputStream in = DecisionTableAnalyzerFromFileTest.class.getResourceAsStream( name );
-        final Reader reader = new InputStreamReader( in );
+    public static String loadResource(final String name) throws
+            Exception {
+        final InputStream in = DecisionTableAnalyzerFromFileTest.class.getResourceAsStream(name);
+        final Reader reader = new InputStreamReader(in);
         final StringBuilder text = new StringBuilder();
         final char[] buf = new char[1024];
         int len = 0;
-        while ( ( len = reader.read( buf ) ) >= 0 ) {
-            text.append( buf,
-                         0,
-                         len );
+        while ((len = reader.read(buf)) >= 0) {
+            text.append(buf,
+                        0,
+                        len);
         }
         return text.toString();
     }
 
-    public static void assertOnlyContains( final Set<Issue> result,
-                                           final String... expected ) {
-
-        if ( result.isEmpty() ) {
-            fail( "Data is empty." );
+    public static void assertOnlyContains(final Set<Issue> result,
+                                          final String... expected) {
+        if (result.isEmpty()) {
+            fail("Data is empty.");
         }
 
-        for ( final Issue issue : result ) {
-            String title = ExplanationProvider.toTitle( issue );
-            if ( !contains( expected,
-                            title ) ) {
-                fail( "Should not find: " + title );
-            }
-        }
-    }
+        List<String> resultTitles = result.stream()
+                .map(issue -> ExplanationProvider.toTitle(issue))
+                .distinct()
+                .collect(Collectors.toList());
```

Comment:
Not serious, so just FYI: `.distinct().collect(Collectors.toList())` should be identical to `.collect(Collectors.toSet())`